src/components/Dashboard.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Header, Button, Icon, SemanticCOLORS, SemanticICONS } from 'semantic-ui-react'
+import { useUserState } from '../contexts/UserState'
+import constants from '../utils/constants'
+import strings from '../utils/constants'
+import { USER_ROLES } from '../utils/data'
+import { PermissionPolicyType, Filter } from '../utils/generated/graphql'
+import useListApplications from '../utils/hooks/useListApplications'
+import useListTemplates from '../utils/hooks/useListTemplates'
+import usePageTitle from '../utils/hooks/usePageTitle'
+import { TemplateDetails, TemplateInList } from '../utils/types'
+
+const Dashboard: React.FC = () => {
+  const {
+    userState: { templatePermissions, isNonRegistered },
+    logout,
+  } = useUserState()
+  const {
+    templatesData: { templatesByCategory },
+  } = useListTemplates(templatePermissions, false)
+  if (isNonRegistered) {
+    logout()
+    return null
+  }
+
+  usePageTitle(strings.PAGE_TITLE_HOME)
+
+  return (
+    <div id="dashboard">
+      {templatesByCategory.map(
+        ({ templates, templateCategory: { categoryIcon, categoryTitle } }) => (
+          <div key={categoryTitle} className="template-category">
+            <div className="title">
+              {categoryIcon && (
+                <Icon size="large" color="grey" name={categoryIcon as SemanticICONS} />
```

Comment:
Also here -- rather than use `as`, it might be better to define the type where categoryIcon is first declared (i.e. useListTemplates line 88):
`let categoryIcon: SemanticICONS`

Reply:
60b783cb8c841ed6744add9dc7c03868b26e540b

Remembered to relocate icon color to style and still need to cast filter.icon, since it's a string type in DB